recipe_scrapers/saltpepperskillet.py

```diff
@@ -8,3 +9,24 @@ def host(cls):
 
     def author(self):
         return self.schema.author().title()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h4",
+            ".wprm-recipe-ingredient",
+        )
+
+    def equipment(self):
+        equipment_container = self.soup.find(
+            "div", class_="wprm-recipe-equipment-container"
+        )
+        if equipment_container:
+            return [
+                item.get_text(strip=True)
+                for item in equipment_container.find_all(
+                    "div", class_="wprm-recipe-equipment-name"
+                )
+            ]
+        return None
```

Comment:
```suggestion
        if not equipment_container:
            return None
        return [
            item.get_text(strip=True)
            for item in equipment_container.find_all(
                "div", class_="wprm-recipe-equipment-name"
            )
        ]
```

...and maybe `get_equipment`? :)